comid/measurement_test.go

```diff
@@ -112,3 +123,227 @@ func TestMeasurement_NewUUIDMeasurement_bad_uuid(t *testing.T) {
 
 	assert.Nil(t, tv.SetUUID(nonRFC4122UUID))
 }
+
+var (
+	goldenMKey  uint64 = 100
+	goldenMKey1 uint64 = 120
+)
+
+func TestUIntMkey_NotSet(t *testing.T) {
+	mkey := &Mkey{}
+	expectedErr := "unknown measurement key type: <nil>"
+	err := mkey.Valid()
+	require.NotNil(t, err)
```

Comment:
This is a test assertion (`assert`) not a prerequisite (`require`).  Anyway, it is also made redundant by the assertion on the following line, so it should be removed.